src/unitxt/schema.py

```diff
@@ -32,7 +33,13 @@ class ToUnitxtGroup(StreamInstanceOperatorValidator):
     postprocessors: List[str] = field(default_factory=lambda: ["to_string_stripped"])
     remove_unnecessary_fields: bool = True
 
+    def _to_lists_of_keys_and_values(self, dict: Dict[str, str]):
+        return {"key": [key for key, _ in dict.items()], "value": [str(value) for _, value in dict.items()]}
```

Comment:
```suggestion
        return {"key": list(dict.keys()), "value": [str(value) for value in dict.values()]}
```
or does that make an issue with the iteration order, that is not guaranteed to be the same for `keys` or `values`?

Reply:
I'm not sure.  Just wanted to keep it simple and consistent.